### DISCUSSION TITLE
Enterprise license  - questions about costs

### DISCUSSION THREAD

**Discussion by timowevel1** [ant-media/Ant-Media-Server#6742] [FAQs] [answered]
Hello,

our client has an enterprise license and contacted the support regarding questions about the pricing. 

Currently, he has one instance deployed with antmedia over the AWS marketplace. 

However, he currently pays for an enterprise license monthly in his account.
<img width="967" alt="image" src="https://github.com/user-attachments/assets/7c5dd8cb-7564-4c0a-a8cb-13df6a288663">

And sometimes he even pays pay-as-you-go usage:
<img width="780" alt="image" src="https://github.com/user-attachments/assets/121e2749-fa05-45e8-9062-9793951b99f2">

I dont find any information about the pay as you go license, this is very intransparent to me. If he pays the AWS instance, where you already pay license costs, how does he pay an enterprice license and pay as you go usage? 

Thanks.

**Comment by muratugureminoglu** [comment, marked as answer]
Hi @timowevel1 

Thanks for your question.

Your ticket was created three days ago on Sunday (We don't work generally on Sundays). The support team has forwarded it to the sales team, and as this review process can take some time, there may be a slight delay.

Could you please share the AMI ID and region you’re using with AWS? This will help me determine whether you’re using the Marketplace or Pay-As-You-Go (Monthly) model.

After reviewing your subscription, I see it’s being used for two instances: one on a $99 monthly plan and another on an hourly (Pay-As-You-Go) basis. I’ll  forward via email with the IP addresses where the license is currently active.

1. 18.175.xxx.xxx/172.31.17.57
2. 3.76.xx.xxx/172.31.45.133


I hope this addresses your questions!
